src/PerformanceTests/Common.Data/Scenarios/ICreateSeedData.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Common.Scenarios
+{
+    using NServiceBus;
+
+    interface ICreateSeedData
+    {
+        int SeedSize { get; set; }
```

Comment:
Yep, made mistake in accessing seedsize in incorrect partial class. Fixed it and added Guard that seedsize was properly set.